lib/view_component/compiler.rb

```diff
@@ -121,13 +126,23 @@ def template_errors
 
     def templates
       @templates ||= begin
-        extensions = ActionView::Template.template_handler_extensions
+        extensions = %w[builder erb haml html jbuilder raw ruby slim]
```

Comment:
Is there a way to achieve this without hardcoding these values?

Reply:
This is just a cosmetic change, the value of `ActionView::Template.template_handler_extensions` is exactly the array added. I think we should try to minimize Rails dependency, so the library could be used from other frameworks... but that's not the objective of this merge, so I removed it.